cg/utils/error_handler.py

```diff
@@ -0,0 +1,20 @@
+"""Module that holds a generic error handling decorator."""
+
+from cg.exc import CgError
+
+
+def handle_errors(to_except: tuple, to_raise):
+    """Handle errors for the Post processing services."""
+
+    def error_handling(fn):
+        def wrapper(*args, **kwargs):
+            try:
+                return fn(*args, **kwargs)
+            except to_except as error:
+                raise to_raise(error) from error
+            except Exception as error:
+                raise CgError(f"{error}") from error
+
+        return wrapper
+
+    return error_handling
```

Comment:
We had a discussion during your vacation about these error handlers and we all liked them, but though the general ones where you pass exceptions to it where were not as general as they seem. I would prefer having this function in the orders module and specifying in the method itself what to catch and raise